lib/muffin_man/enable_logger.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module MuffinMan
+  module EnableLogger
+    LOGGING_ENABLED = true
+
+    def log_request_and_response(level, res)
+      log_info = "REQUEST\n
+        canonical_uri:#{canonical_uri}\n\n
+        query_params:#{query_params}\n\n
+        RESPONSE\n
+        response_headers=#{res.headers}\n\n
+        response_body=#{res.body}\n\n
```

Comment:
Please conform all secured inputs are not logged

Reply:
headers will be observed on staging when deployed for QA

Reply:
Can we confirm this without merging? I would prefer this not be merged into the gem unless we know it's not logging secured inputs.